### ISSUE TITLE
WE no GCD skills visual bug ? 

### ISSUE THREAD

**Issue by AkameSins** [WarEmu/WarBugs#19159]
~ The disappearance of the cooldown on abilities without a global cooldown, such as elixir of shadows or feinted position needs to be fixed as well.

https://youtu.be/_g4QeFfUG_s

As you can see here when the skills hows its available, I spam it and it still takes like 4-5 seconds to be activated. Is there something I am missing and dont understand ot just a visual bug ? Same happens with Feinted Position. Also in RvR Big fights the cooldown numbers some times never appear.

**Comment by lilimx** [comment]
Hey,

so I've tried to reproduce it. It happend once when suddenly a 5 sec CD started.

**Comment by AkameSins** [comment]
So that means there is a weird bug going on, or something I dont understand? :P

**Comment by lilimx** [comment]
It's something we should keep an eye on it...if you know more about the issue, put it in here :D

**Comment by Symbelmyne** [comment]
It looks like desync of skill`s cooldown with server ticks. Can it be addon like Textured Buttons? I assume you talking about CD went off elixir but skill is still not available to cast for next few seconds, right?

**Comment by AkameSins** [comment]
> It looks like desync of skill`s cooldown with server ticks. Can it be addon like Textured Buttons? I assume you talking about CD went off elixir but skill is still not available to cast for next few seconds, right?

Yes after it was back for the next 4-5 seconds it couldnt be used.
